Library/Homebrew/os/mac.rb

```diff
@@ -214,6 +214,7 @@ def preferred_arch
       "6.3"   => { :clang => "6.1", :clang_build => 602 },
       "6.3.1" => { :clang => "6.1", :clang_build => 602 },
       "6.3.2" => { :clang => "6.1", :clang_build => 602 },
+      "7.0"   => { :clang => "6.1", :clang_build => 602 },
```

Comment:
Yeah, exactly - the clang I see is labelled 7.0.0 build 700, so assuming it's the same in Xcode, we want `{ :clang => "7.0", :clang_build => 700 }`